airflow/example_dags/docker_copy_data.py

```diff
@@ -0,0 +1,84 @@
+'''
+This sample "listen to directory". move the new file and print it, using docker-containers.
+The following operators are being used: DockerOperator, BashOperator & ShortCircuitOperator.
+TODO: Review the workflow, change it accordingly to to your environment & enable the code.
+'''
+
+# from __future__ import print_function
```

Comment:
@criccomini it works like a charm :-)
You just need to adapt it to your environment, for example you might change the following settings:
* docker_url - replace it with swarm/docker endpoint
* volumes - '/your/host/input_dir/path' & etc.